devel/Makefile.am

```diff
@@ -35,9 +35,10 @@ EXTRA_DIST		= $(EXTRA_SCRIPTS) \
 #
 # ref-passed-variables-inited.cocci seems to be returning some false positives around
 # GHashTableIters, so it is disabled for the moment.
-COCCI_FILES ?= coccinelle/string-any-of.cocci \
-			   coccinelle/string-empty.cocci \
-			   coccinelle/string-null-matches.cocci
+COCCI_FILES ?=	coccinelle/string-any-of.cocci			\
+		coccinelle/string-empty.cocci			\
+		coccinelle/string-null-matches.cocci		\
+		coccinelle/use-func.cocci
```

Comment:
You'll need to add this to EXTRA_DIST too.  It would be nice to just include reference COCCI_FILES in there so we don't have to add things to two places.

Reply:
Ah, will do